models/comment_test.go

```diff
@@ -25,23 +23,17 @@ func TestWeCanStructureCommentsCorrectly(t *testing.T) {
 	setupTestingDB()
 
 	// setup users
-	CreateUser("person@example.com", "friendlyPerson", "password")
-	CreateUser("troll@example.com", "uglyTroll", "password")
-	CreateUser("moderator@example.com", "uglyTroll", "password")
-	person := User{}
-	person.FindByEmail("person@example.com")
-	troll := User{}
-	troll.FindByEmail("troll@example.com")
-	moderator := User{}
-	moderator.FindByEmail("moderator@example.com")
+	person, _ := UserFactory("person@example.com", "friendlyPerson", "password")
+	troll, _ := UserFactory("troll@example.com", "uglyTroll", "password")
+	moderator, _ := UserFactory("moderator@example.com", "uglyTroll", "password")
 
 	// create a post and a typical comment thread
 	post, _ := person.CreatePost("Post Title", "post-title", "lorem ipsum", "image")
-	nastyComment, _ := troll.CommentOnPost(post, nil, "text", "WOW, this post is dumb!", "")
-	reply, _ := person.CommentOnPost(post, &nastyComment, "text", "Hay, I worked really hard on this", "")
-	nastyAgain, _ := troll.CommentOnPost(post, &reply, "text", "Don't you mean \"Hey\"? You must be as dumb as the post you created.", "")
-	moderatorComment, _ := moderator.CommentOnPost(post, nil, "text", "Hey troll, play nice or go somewhere else. We may shut down commenting if you keep this up.", "")
-	trollIsAskingForIt, _ := troll.CommentOnPost(post, &moderatorComment, "text", "You're dumb too!", "")
+	nastyComment, _ := troll.CommentOnPost(post, nil, "text", "WOW, this post is dumb!")
+	reply, _ := person.CommentOnPost(post, &nastyComment, "text", "Hay, I worked really hard on this")
+	nastyAgain, _ := troll.CommentOnPost(post, &reply, "text", "Don't you mean \"Hey\"? You must be as dumb as the post you created.")
+	moderatorComment, _ := moderator.CommentOnPost(post, nil, "text", "Hey troll, play nice or go somewhere else. We may shut down commenting if you keep this up.")
+	trollIsAskingForIt, _ := troll.CommentOnPost(post, &moderatorComment, "text", "You're dumb too!")
```

Comment:
these changes are unrelated to the refactor for the user creation, but because we have a new migration in master the test suite needs this change to pass.